mathics/doc/common_doc.py

```diff
@@ -823,6 +817,9 @@ def __init__(self, want_sorting=False):
                             if not builtin.__class__.__name__.endswith("Box")
                         ]
                         for instance in subsections:
+                            if hasattr(instance, "no_doc") and instance.no_doc:
```

Comment:
This code can be simplified if we ensure a "no_doc" or better, "add_doc"  (or suggest something without "no_") , is added to a base Builtin class somewhere. 

Let me know your thoughts.